e2e/infra/ui/browserWrapper.ts

```
@@ -0,0 +1,62 @@
+    }
+
+    async navigateTo(url: string) {
+        if (!this.page) {
+            throw new Error('Browser is not launched yet!');
+        }
+        await this.page.goto(url);
+        await this.page.waitForLoadState('networkidle');
+    }
+      
+    async closePage() {
+        this.page ? await this.page.close() : this.page = null;
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)